### PR TITLE
add Chinese document for introduction what-is-kubesphere page

### PR DESCRIPTION
**Pull request by Jaycean**
- add introduction what-is-kubesphere page

### CODE REVIEW

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 1]
```diff
@@ -1,60 +1,60 @@
 ---
-title: "What is KubeSphere"
+title: "什么是 KubeSphere"
 keywords: 'Kubernetes, KubeSphere, Introduction'
```
Comment (Sherlock113) [inline content/zh/docs/introduction/what-is-kubesphere.md:3]: Please translate values in keywords as well. Here, only Introduction needs to be translated.

->
keywords: 'Kubernetes, KubeSphere, 介绍'

**Review by Sherlock113 — COMMENTED** [review]
Please have a look at my comments and make the change. Thanks.

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 2]
```diff
@@ -1,60 +1,60 @@
...
+description: '什么是 KubeSphere'
 
 weight: 1100
 ---
 
-## Overview
+## 概述
 
-[KubeSphere](https://kubesphere.io) is a **distributed operating system managing cloud-native applications** with [Kubernetes](https://kubernetes.io) as its kernel, providing a plug-and-play architecture for the seamless integration of third-party applications to boost its ecosystem.
+[KubeSphere](https://kubesphere.io) 是在[Kubernetes](https://kubernetes.io)之上构建的面向云原生应用的**容器混合云**，支持多云与多集群管理，提供全栈的 IT 自动化运维的能力，简化企业的 DevOps 工作流。它的架构可以很方便地与云原生生态进行即插即用 (plug-and-play) 的集成。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 3]
```diff
@@ -1,60 +1,60 @@
...
 
-KubeSphere also represents a multi-tenant enterprise-grade container platform with full-stack automated IT operation and streamlined DevOps workflows. It provides developer-friendly wizard web UI, helping enterprises to build out a more robust and feature-rich platform. It boasts the most common functionalities needed for enterprise Kubernetes strategies, such as Kubernetes resource management, DevOps (CI/CD), application lifecycle management, monitoring, logging, service mesh, multi-tenancy, alerting and notification, auditing, storage and networking, autoscaling, access control, GPU support, multi-cluster deployment and management, network policy, registry management, and security management.
+KubeSphere 提供了全栈化容器部署与多租户管理平台。 它提供了运维友好的向导式操作界面，帮助企业快速构建一个强大和功能丰富的容器云平台。它拥有 Kubernetes 企业级服务所需的最常见功能，例如 Kubernetes 资源管理、DevOps、微服务治理、日志查询与收集、灰度发布、服务与网络、多租户、监控告警、审计、存储和网络、自动定量、访问控制、GPU的支持、多集群部署和管理、网络政策、注册表管理和安全管理等多种业务场景。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 4]
```diff
@@ -1,60 +1,60 @@
...
 
-KubeSphere delivers **consolidated views while integrating a wide breadth of ecosystem tools** around Kubernetes, thus providing consistent user experiences to reduce complexity. At the same time, it also features new capabilities that are not yet available in upstream Kubernetes, alleviating the pain points of Kubernetes including storage, network, security and usability. Not only does KubeSphere allow developers and DevOps teams use their favorite tools in a unified console, but, most importantly, these functionalities are loosely coupled with the platform since they are pluggable and optional.
+KubeSphere **在围绕 Kubernetes 集成各种生态系统工具的同时** ，提供了一致的用户体验以降低复杂性。 同时，它还提供了 Kubernetes 尚未提供的新功能，旨在解决 Kubernetes 本身存在的存储、网络、安全和易用性等痛点。KubeSphere 不仅允许开发人员和 DevOps 团队在统一的控制台中使用他们喜欢的工具，而且最重要的是，这些功能与平台是松散耦合的，因为它们是可插入的和可选的。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 5]
```diff
@@ -1,60 +1,60 @@
...
+作为一个轻量级平台，KubeSphere 对不同的云生态系统支持变得更加友好，因为它根本不会改变 Kubernetes 本身功能。 换句话说，KubeSphere 可以 **部署和运行在任何基础架构与任何现有的版本兼容的 Kubernetes 集群上** ，包括虚拟机，裸机，本地部署，公有云和混合云。 KubeSphere 用户可以选择在云和容器平台（例如阿里云，AWS，QingCloud，腾讯云，华为云 和 Rancher）上安装 KubeSphere，甚至可以导入和管理使用 Kubernetes 发行版创建的现有 Kubernetes 集群。 KubeSphere 可以在不修改其当前资源或资产、不影响用户的业务的情况下与现有 Kubernetes 平台的无缝集成。有关更多信息，请参见[安装](../../installing-on-linux/)。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 6]
```diff
@@ -1,60 +1,60 @@
...
+## 3.0 新增功能
 
-- **Multi-cluster Management**. As we usher in an era of hybrid cloud, multi-cluster management has emerged as the call of our times. It represents one of the most necessary features on top of Kubernetes as it addresses the pressing need of our users. In the latest version 3.0, we have equipped KubeSphere with its unique multi-cluster feature that is able to provide a central control plane for clusters deployed in different clouds. Users can import and manage their existing Kubernetes clusters created on the platform of mainstream infrastructure providers (e.g. Amazon EKS and Google Kubernetes Engine). This will greatly reduce the learning cost for our users with operation and maintenance process streamlined as well. Solo and Federation are the two featured patterns for multi-cluster management, making KubeSphere stand out among its counterparts.
+- **多集群管理**： 随着我们迎来混合云时代，多集群管理已成为我们时代的呼唤。 它代表了 Kubernetes 上最必要的功能之一，它可以满足用户的迫切需求。 在最新版本 3.0 中，我们为 KubeSphere 配备了多集群功能，该功能可以为部署在不同云中的集群提供中央控制操作界面。 用户可以导入和管理在主流基础架构提供商（例如 Amazon EKS 和 Google Kubernetes Engine）平台上创建的现有 Kubernetes 集群。 通过简化操作和维护流程，这将大大降低我们用户的学习成本。 Solo 和 Federation 是多集群管理的两个特色模式，使KubeSphere 在同类软件中脱颖而出。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 7]
```diff
@@ -1,60 +1,60 @@
...
+  - **网络策略和隔离**： 网络策略允许在同一群集内进行网络隔离，这意味着可以在某些实例（Pods）之间设置防火墙。 通过配置网络隔离来控制同一群集中 Pod 之间的流量和来自外部的流量，用户可以隔离具有增强安全性的应用程序。 他们还可以决定是否可以从外部访问服务。
 
-  - **Open Policy Agent**. KubeSphere provides flexible, fine-grained access control based on [Open Policy Agent](https://www.openpolicyagent.org/). Users can manage their security and authorization policies in a unified way with a general architecture.
+  - **Open Policy Agent**： KubeSphere 提供了细粒度的访问控制： [Open Policy Agent](https://www.openpolicyagent.org/)。 用户可以使用通用体系结构以统一的方式管理其安全性和授权策略。
 
-  - **OAuth 2.0**. Users can now easily integrate third-party applications with OAuth 2.0 protocol.
+  - **OAuth 2.0**： 用户现在可以轻松地将第三方应用程序与 OAuth 2.0 协议集成在一起。
 
-- **Multilingual Support of Web Console**. KubeSphere is designed for users around the world at the very beginning. Thanks to our community members across the globe, KubeSphere 3.0 now supports four official languages for its web console: English, Simplified Chinese, Traditional Chinese, and Spanish. More languages are expected to be supported going forward.
+- **操作控制台的多语言支持**： KubeSphere 最初是为全球用户设计的。 感谢全球的社区成员，KubeSphere 3.0 现在支持其 Web 控制台的四种官方语言：英语，简体中文，繁体中文和西班牙语。 预计将来将支持更多语言。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 8]
```diff
@@ -1,60 +1,60 @@
...
 
-In addition to the above highlights, KubeSphere 3.0 also features other functionality upgrades. For more and detailed information, see [Release Notes for 3.0.0](../../release/release-v300/).
+除了上述重点之外，KubeSphere 3.0 还具有其他功能升级。 有关更多详细信息，请参阅 3.0.0 的发行说明。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 9]
```diff
@@ -1,60 +1,60 @@
...
-## Roadmap
+## 产品规划
 
-### Express Edition -> KubeSphere 1.0.x -> KubeSphere 2.0.x -> KubeSphere 2.1.x -> KubeSphere 3.0.0
+### 易捷版 -> KubeSphere 1.0.x -> KubeSphere 2.0.x -> KubeSphere 2.1.x -> KubeSphere 3.0.0
 
 ![Roadmap](https://pek3b.qingstor.com/kubesphere-docs/png/20190926000413.png)
 
-## Landscape
+## 前景展望
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md (hunk 1 above)
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done.
Roger, I will pay attention to the follow-up translation

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 10]
```diff
@@ -1,60 +1,60 @@
...
+- **操作控制台的多语言支持**： KubeSphere 最初是为全球用户设计的。 感谢全球的社区成员，KubeSphere 3.0 现在支持其 Web 控制台的四种官方语言：英语，简体中文，繁体中文和西班牙语。 预计将来将支持更多语言。
 
-In addition to the above highlights, KubeSphere 3.0 also features other functionality upgrades. For more and detailed information, see [Release Notes for 3.0.0](../../release/release-v300/).
+除了上述重点之外，KubeSphere 3.0 还具有其他功能升级。 有关更多详细信息，请参阅 3.0.0 的发行说明。
 
-## Open Source
+## 开源
 
-As we adopt the open source model, development is proceeding in an open way and driven by KubeSphere community. KubeSphere is **100% open source** and available on [GitHub](https://github.com/kubesphere/) where you can find all the source code, documents and discussions. It has been widely installed and used in development, testing and production environments, and a large number of services are running smoothly in KubeSphere.
+KubeSphere 所有版本都 **100% 开源免费** ，已大规模服务于社区用户，广泛地应用在以 Docker 和 Kubernetes 为中心的开发测试及生产环境，大量服务平稳地运行在 KubeSphere 之上。您可在 [GitHub](https://github.com/kubesphere/) 上找到所有源代码，文档和讨论。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

**Comment by Sherlock113** [comment]
截屏2020-09-23下午6 30 17
Please click load more. Some changes not made yet.

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 11]
```diff
@@ -1,60 +1,60 @@
...
+作为一个轻量级平台，KubeSphere 对不同的云生态系统支持变得更加友好，因为它根本不会改变 Kubernetes 本身功能。 换句话说，KubeSphere 可以 **部署和运行在任何基础架构与任何现有的版本兼容的 Kubernetes 集群上** ，包括虚拟机，裸机，本地部署，公有云和混合云。 KubeSphere 用户可以选择在云和容器平台（例如阿里云，AWS，QingCloud，腾讯云，华为云 和 Rancher）上安装 KubeSphere，甚至可以导入和管理使用 Kubernetes 发行版创建的现有 Kubernetes 集群。 KubeSphere 可以在不修改其当前资源或资产、不影响用户的业务的情况下与现有 Kubernetes 平台的无缝集成。有关更多信息，请参见[安装](../../installing-on-linux/)。
 
-KubeSphere screens users from the infrastructure underneath and helps enterprises modernize, migrate, deploy and manage existing and containerized apps seamlessly across a variety of infrastructure types. This is how KubeSphere empowers developers and Ops teams to focus on application development and accelerate DevOps automated workflows and delivery processes with enterprise-level observability and troubleshooting, unified monitoring and logging, centralized storage and networking management, easy-to-use CI/CD pipelines, and so on.
+KubeSphere 从基础架构中选择用户，并帮助企业跨各种基础架构类型无缝地现代化、迁移、部署和管理现有的容器化的应用程序。 通过这种方式，KubeSphere 使开发人员和操作团队能够专注于应用程序开发，并通过企业级可观察性和故障排除、统一监控和日志记录、集中式存储和网络管理、易于使用的 CI/CD 管道来加快 DevOps 自动化工作流和交付流程等等。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 12]
```diff
@@ -1,60 +1,60 @@
...
+- **多集群管理**： 随着我们迎来混合云时代，多集群管理已成为我们时代的呼唤。 它代表了 Kubernetes 上最必要的功能之一，它可以满足用户的迫切需求。 在最新版本 3.0 中，我们为 KubeSphere 配备了多集群功能，该功能可以为部署在不同云中的集群提供中央控制操作界面。 用户可以导入和管理在主流基础架构提供商（例如 Amazon EKS 和 Google Kubernetes Engine）平台上创建的现有 Kubernetes 集群。 通过简化操作和维护流程，这将大大降低我们用户的学习成本。 Solo 和 Federation 是多集群管理的两个特色模式，使KubeSphere 在同类软件中脱颖而出。
 
-- **Improved Observability**. We have enhanced observability as it becomes more powerful to include custom monitoring, tenant event management, diversified notification methods (e.g. WeChat and Slack) and more features. Among others, users can now customize monitoring dashboards, with a variety of metrics and graphs to choose from for their own needs. It also deserves to mention that KubeSphere 3.0 is compatible with Prometheus, which is the de facto standard for Kubernetes monitoring in the cloud-native industry.
+- **改善可观察性**： 随着自定义监控，租户事件管理，多样化的通知方法（例如，微信和 Slack）以及更多功能变得更加强大，我们提高了可观察性。 另外，用户现在可以自定义监视仪表板，并根据自己的需要选择各种度量和图形。 还值得一提的是，KubeSphere 3.0 与 Prometheus 兼容，后者是云原生行业中 Kubernetes 监控的业内标准。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 13]
```diff
@@ -1,60 +1,60 @@
...
 
-- **Enhanced Security**. Security has alway remained one of our focuses in KubeSphere. In this connection, feature enhancements can be summarized as follows:
+- **增强安全性**：安全始终是我们在KubeSphere中关注的重点之一。 在这方面，功能增强可以概括如下：
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 14]
```diff
@@ -1,60 +1,60 @@
...
 
-  - **Auditing**. Records will be kept to track who does what at what time. The support of auditing is extremely important especially for traditional industries such as finance and banking.
+  - **审计**： 操作记录将被保存，以跟踪谁在什么时间做什么。 审计的支持极为重要，特别是对于金融和银行业等传统行业而言。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 15]
```diff
@@ -1,60 +1,60 @@
...
 
-  - **Network Policy and Isolation**. Network policies allow network isolation within the same cluster, which means firewalls can be set up between certain instances (Pods). By configuring network isolation to control traffic among Pods within the same cluster and traffic from outside, users can isolate applications with security enhanced. They can also decide whether services are accessible externally.
+  - **网络策略和隔离**： 网络策略允许在同一群集内进行网络隔离，这意味着可以在某些实例（Pods）之间设置防火墙。 通过配置网络隔离来控制同一群集中 Pod 之间的流量和来自外部的流量，用户可以隔离具有增强安全性的应用程序。 他们还可以决定是否可以从外部访问服务。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 16]
```diff
@@ -1,60 +1,60 @@
...
 
-  - **Open Policy Agent**. KubeSphere provides flexible, fine-grained access control based on [Open Policy Agent](https://www.openpolicyagent.org/). Users can manage their security and authorization policies in a unified way with a general architecture.
+  - **Open Policy Agent**： KubeSphere 提供了细粒度的访问控制： [Open Policy Agent](https://www.openpolicyagent.org/)。 用户可以使用通用体系结构以统一的方式管理其安全性和授权策略。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 17]
```diff
@@ -1,60 +1,60 @@
...
 
-  - **OAuth 2.0**. Users can now easily integrate third-party applications with OAuth 2.0 protocol.
+  - **OAuth 2.0**： 用户现在可以轻松地将第三方应用程序与 OAuth 2.0 协议集成在一起。
```
[the remark this answers is not in the document]
Comment (Jaycean) [reply content/zh/docs/introduction/what-is-kubesphere.md]: done

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 18]
```diff
@@ -1,60 +1,60 @@
...
+作为全栈化容器部署与多租户管理平台，KubeSphere 提供了运维友好的向导式操作界面，帮助企业快速构建一个强大和功能丰富的容器云平台。它拥有 Kubernetes 企业级服务所需的最常见功能，例如 Kubernetes 资源管理、DevOps、多集群部署与管理、应用生命周期管理、微服务治理、日志查询与收集、服务与网络、多租户管理、监控告警、事件审计、存储、访问控制、GPU 支持、网络策略、镜像仓库管理以及安全管理等。
 
-KubeSphere delivers **consolidated views while integrating a wide breadth of ecosystem tools** around Kubernetes, thus providing consistent user experiences to reduce complexity. At the same time, it also features new capabilities that are not yet available in upstream Kubernetes, alleviating the pain points of Kubernetes including storage, network, security and usability. Not only does KubeSphere allow developers and DevOps teams use their favorite tools in a unified console, but, most importantly, these functionalities are loosely coupled with the platform since they are pluggable and optional.
+KubeSphere **围绕 Kubernetes 集成了各种生态系统的工具，** ，提供了一致的用户体验以降低复杂性。同时，它还具备 Kubernetes 尚未提供的新功能，旨在解决 Kubernetes 本身存在的存储、网络、安全和易用性等痛点。KubeSphere 不仅允许开发人员和 DevOps 团队在统一的控制台中使用他们喜欢的工具，而且最重要的是，这些功能与平台松散耦合，因为他们可以选择是否安装这些可拔插组件。
 
-## Run KubeSphere Everywhere
+## 随处运行 KubeSphere
```
Comment (FeynmanZhou) [inline content/zh/docs/introduction/what-is-kubesphere.md:17]: 支持在任意平台运行

File: content/zh/docs/introduction/what-is-kubesphere.md [hunk 19]
```diff
@@ -1,60 +1,60 @@
...
+作为一个轻量级平台，KubeSphere 对不同云生态系统的支持变得更加友好，因为它根本不会改变 Kubernetes 本身的功能。换句话说，KubeSphere 可以 **部署并运行在任何基础架构以及所有兼容现有版本的 Kubernetes 集群上** ，包括虚拟机、裸机、本地环境、公有云和混合云等。KubeSphere 用户可以选择在云和容器平台（例如阿里云、AWS、青云QingCloud、腾讯云、华为云和 Rancher 等）上安装 KubeSphere，甚至可以导入和管理使用 Kubernetes 发行版创建的现有 Kubernetes 集群。KubeSphere 可以在不修改用户当前的资源或资产、不影响其业务的情况下与现有 Kubernetes 平台无缝集成。有关更多信息，请参见[在 linux 上安装](../../installing-on-linux/)和[在 Kubernetes 上安装](../../installing-on-kubernetes/)。
```
Comment (FeynmanZhou) [inline content/zh/docs/introduction/what-is-kubesphere.md:19]: 因为它根本不会改变 Kubernetes 本身的功能 => 因为它没有对 Kubernetes 本身有任何的 Hack

**Pull request merged**